app/models/file_search.rb

```diff
@@ -19,6 +19,23 @@ def venue_names
     params.fetch(:venues) { [] }
   end
 
+  def self.year_range_limit
+    1935..Date.today.year
```

Comment:
Just a reminder about the potential Time Zone bug. `Time.zone.now.year` might avoid it.